Documentation/HPC-Packaging.md

```diff
@@ -166,11 +239,49 @@ are required (anything in [..] is optional):
     files. They should be created from within the spec file (ie build script)
     using a 'here-style' document:
     ```
```

Comment:
Unfortunately, my comment to this got added to the wrong line last time so it was out of context - but it is valid regardless:
The module file content below is just an example. This should be noted.
Otherwise, we'd stuff it into a macro.